.github/workflows/crio.yml

```diff
@@ -93,24 +26,20 @@ jobs:
         run: |
           echo "::set-env name=GOPATH::${{ github.workspace }}"
           echo "::add-path::${{ github.workspace }}/bin"
-
       - name: Setup system
         run: |
           # enable necessary kernel modules
           sudo ip6tables --list >/dev/null
-
```

Comment:
I personally prefer to keep some optical separation (not blocking).

Reply:
Totally agree. Something seemed to get messed up in splitting the files. It is addressed now.